src/common.speech/DialogConnectorFactory.ts

```diff
@@ -66,18 +66,25 @@ export class DialogConnectionFactory extends ConnectionFactoryBase {
         const {resourcePath, version, authHeader} = getDialogSpecificValues(dialogType);
 
         const headers: IStringDictionary<string> = {};
-        headers[authInfo.headerName] = authInfo.token;
+        if (authInfo.token !== undefined && authInfo.token !== "") {
```

Comment:
don't you have to check for null as well since it comes from the API developer?

Reply:
Good call, replacing with a "juggling-check"